src/ecs-additions/ecs-additions-cromwell-linux2-worker.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
```

Comment:
Now that all the solutions use the same mechanism for autoscaling container disk space, we could probably collapse everything into one script.

Reply:
That's actually a good idea although right now Cromwell with Batch workers on AL1 doesn't use this script at all as the commands are specific to AL2

Reply:
👍